pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -6,22 +6,26 @@ WARNING: DO NOT edit .pxi FILE directly, .pxi is generated from .pxi.in
 
 {{py:
 
-# dtype, ttype, c_type
-dtypes = [('float64', 'float64', 'float64_t'),
-          ('float32', 'float32', 'float32_t'),
-          ('uint64', 'uint64', 'uint64_t'),
-          ('uint32', 'uint32', 'uint32_t'),
-          ('uint16', 'uint16', 'uint16_t'),
-          ('uint8', 'uint8', 'uint8_t'),
-          ('object', 'pymap', 'object'),
-          ('int64', 'int64', 'int64_t'),
-          ('int32', 'int32', 'int32_t'),
-          ('int16', 'int16', 'int16_t'),
-          ('int8', 'int8', 'int8_t')]
+# dtype, ttype, c_type, to_c_type, to_dtype
+dtypes = [('complex128', 'complex128', 'khcomplex128_t', \
+                         "to_khcomplex128_t", "to_complex128"),
+          ('complex64', 'complex64', 'khcomplex64_t', \
```

Comment:
i think backslash is unnecessary.

can you stick with either all-single-quotes or all-double quotes

Reply:
Thanks,  done.